packages/app/background-jobs-common/src/background-job-processor/types.ts

```diff
@@ -20,28 +20,6 @@ export type SafeJob<T = any, R = any, N extends string = string> = Omit<
   waitUntilFinished(queueEvents: any, ttl?: number): Promise<R>
 }
 
-export type SafeQueue<
```

Comment:
this is no longer necessary, BullMQ changes finally make types between pro and free versions more compatible out-of-the-box